jdenticon/jdenticon-tests.ts

```diff
@@ -0,0 +1,11 @@
+import {jdenticon} from "./index";
```

Comment:
`import jdenticon = require("jdenticon");` style.
this package uses CommonJS style module.
https://jdenticon.com/#does-jdenticon-run-on-node